modules/etc/smartics/camel-modules.xml

```diff
@@ -433,6 +437,38 @@
             <module name="javax.api"/>
             <module name="org.osgi.core"/>
             <module name="org.slf4j"/>
+            <module name="rhino" export="true">
```

Comment:
Is it really true that component mqtt needs to see rhino, python, jruby, groovy. I don't see these dependencies [here](https://github.com/apache/camel/blob/master/components/camel-mqtt/pom.xml)